TestApp/android/app/src/main/assets/appcenter-config.json

```diff
@@ -2,4 +2,4 @@
     "app_secret": "32fcfc69-d576-41dc-8d49-4be159e3d7b2",
     "auth_provider": "AADB2C",
     "enable_push_in_javascript": true
-}
+}
```

Comment:
leave a new line at the end of file. I guess changes to this file are unintentional. so revert it back.

Reply:
got it, will do that.